src/views/CreateUserAccountView/CreateUserAccountPage.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+
+import { Redirect, useHistory } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import {
+  Box,
+  CircularProgress,
+  Container,
+  Typography,
+} from '@material-ui/core';
+import { toast } from 'react-toastify';
+import { useQuery } from '../../old/lib/hooks/useQuery';
+import { langTokens } from '../../locales/localizationInit';
+import { BasicButton, BasicInput } from '../../components/Form';
+import { passwordValidationObj } from '../../old/lib/components/Users/validationRules';
+import { useStyles } from './CreateUserAccountPage.style';
+import { activateUser } from '../../old/lib/utilities/API/api';
+import { LoadingStatusEnum, LoadingStatusType } from '../../old/lib/types';
+
+interface ICreateUserAccount {
+  newPassword: string;
+  matchPassword: string;
+}
+
+const CreateUserAccountPage = () => {
+  const query = useQuery();
+  const history = useHistory();
+  const { t } = useTranslation();
+
+  const classes = useStyles();
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, handleSubmit, errors, watch, formState } = useForm<
```

Comment:
what is ESLint error for `unbound-method` rule here?

Reply:
ESLint: 
Avoid referencing unbound methods which may cause unintentional scoping of `this`.
If your function does not access `this`, you can annotate it with `this: void`, or consider using an arrow function instead.

Just make with example found somewhere in our code